http/server.lua

```diff
@@ -1321,6 +1374,36 @@ local function httpd_start(self)
     return self
 end
 
+-- validate_ssl_opts validates for ssl_opts and return true if at least ssl_cert_file
+-- and ssl_key_file parameteres are not nil.
+local function validate_ssl_opts(ssl_opts)
```

Comment:
It's just shorter and less repeats. 
```suggestion
local function validate_ssl_opts(opts)
```

Reply:
Edited.